index_ivf.go

```diff
@@ -46,13 +46,3 @@ func (idx *IndexImpl) GetSubIndex() (*IndexImpl, error) {
 
 	return &IndexImpl{&faissIndex{subIdx}}, nil
 }
-
-// pass nprobe to be set as index time option for IVF indexes only.
-// varying nprobe impacts recall but with an increase in latency.
-func (idx *IndexImpl) SetNProbe(nprobe int32) {
-	ivfPtr := C.faiss_IndexIVF_cast(idx.cPtr())
-	if ivfPtr == nil {
-		return
-	}
-	C.faiss_IndexIVF_set_nprobe(ivfPtr, C.ulong(nprobe))
```

Comment:
Ack.

Reply:
because 
in 
IndexIVF_c.h there is 
`FAISS_DECLARE_GETTER_SETTER(SearchParametersIVF, size_t, nprobe)`
so it expects nprobe to be a `size_t` which is platform specific
i had done something similar earlier (conversion of int to size_t)